vaadin-spreadsheet-flow-parent/vaadin-spreadsheet-flow/src/main/java/com/vaadin/flow/component/spreadsheet/SpreadsheetTable.java

```diff
@@ -201,4 +205,43 @@ public Collection<PopupButton> getPopupButtons() {
         return Collections.unmodifiableCollection(popupButtons.values());
     }
 
+    /**
+     * Sets the CTAutoFilter object that represents this table in the underlying
+     * POI model.
+     *
+     * @param ctWorksheetAutoFilter
+     *            Referenced autofilter.
+     */
+    public void setCtWorksheetAutoFilter(CTAutoFilter ctWorksheetAutoFilter) {
```

Comment:
I am thinking about making these setters either package private or protected. I can't see the reason for having these as public API. WDYT?

Reply:
Yeah... I think that having getter public is useful, but setters can be protected...

Reply:
--> added a commit which changes the visiblity